src/cf-atomic-component/src/utilities/config.js

```diff
@@ -0,0 +1,51 @@
+/* ==========================================================================
+   Atomic configurations and constants
+
+   ========================================================================== */
+
+'use strict';
+
+// Bit values intended to be used for bit inversion.
+const DIRECTIONS = {
+  UP: 0,
```

Comment:
Should all these values be left aligned to match formatting in other files?

Reply:
Fixed